scripts/test-upgrade.sh

```diff
@@ -37,11 +37,26 @@ RPCD_DIR="$BASE_DIR/rpcd"
 rm /etc/openstack_deploy/user_extra_variables.yml /etc/openstack_deploy/user_variables.yml
 
 # TASK #1
-# Bug: https://github.com/rcbops/u-suk-dev/issues/199
+# Bug:   https://github.com/rcbops/u-suk-dev/issues/199
 # Issue: To avoid any dependency issues that have occured in the past, the
 #        repo_server containers are re-built. This was done as part of the
 #        OSA upgrade script in liberty, but is not done in the OSA mitaka
 #        upgrade script. Thus, we destroy the containers here.
 cd ${OA_DIR}/playbooks
 openstack-ansible lxc-containers-destroy.yml --limit repo_all
 openstack-ansible setup-hosts.yml --limit repo_all
+
+# TASK #3
+# Bug:   https://github.com/rcbops/u-suk-dev/issues/374
+# Issue: Neutron has no migration to correctly set the MTU on existing networks
+#        created in liberty or below.  This work-around sets the MTU on these
+#        networks before the upgrade starts so that instances booted on these
+#        networks after the upgrade has finished will have their MTUs set
+#        correctly.
+# NOTE:  In newton, MTUs will be calculated on the fly and this mtu field will
+#        get dropped.
+VLAN_FLAT="UPDATE networks SET mtu='1500' WHERE id IN (SELECT network_id FROM ml2_network_segments WHERE network_type IN ('vlan','flat'));"
+VXLAN="UPDATE networks SET mtu='1450' WHERE id IN (SELECT network_id FROM ml2_network_segments WHERE network_type='vxlan');"
+
+ansible galera_all[0] -m shell -a "mysql --verbose -e \"${VLAN_FLAT}\" neutron"
+ansible galera_all[0] -m shell -a "mysql --verbose -e \"${VXLAN}\" neutron"
```

Comment:
As per https://github.com/rcbops/u-suk-dev/issues/374:

> Will need detailed documentation. No plans to automate this in pre upgrade tasks, and will be handle by support in their process.

If we do in fact want to automate this, I think a proper neutron script that tries to do the calculation based on physical MTU in neutron.conf would be more appropriate.  That would be outside the scope of this card though.